### DISCUSSION TITLE
Not receiving payout

### DISCUSSION THREAD

**Discussion by leandrofars** [community/community#79104] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I've received a sponsorship on Jul 31, and have not received the payout since then, can someone help?

**Comment by AidenJamesYt** [comment]
[Githubs support article on sponsor payouts](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/paying-for-github-sponsors-by-invoice)


1. Check the payout schedule:

Github sponsorships payout monthly, typically within the first 5 business days of the month following the month the sponsorship was earned. So, in your case, you would have expected the payout sometime between August 1st and 5th.
Check the Github sponsorships documentation for more information on payout timing: https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/paying-for-github-sponsors-by-invoice

2. Verify your payment information:

Make sure your payout method (PayPal or Stripe) is still valid and has not expired.
Ensure your payout information within your Github settings is accurate and up-to-date.

3. Review the sponsorship details:

Check if there are any minimum payout thresholds set by the sponsor.
Some sponsors may require you to reach a certain amount of earnings before releasing a payout.

4. Contact Github Support via a ticket

If you've confirmed everything above and still haven't received your payout by the first week of the month following the month of the sponsorship, it's best to contact Github Support directly. They can investigate the issue further and provide more specific assistance: https://support.github.com/

5. Consider contacting your sponsor:

While not ideal, you can also reach out to your sponsor directly and politely inquire about the payout status. This can be helpful if they have any specific requirements or processes for releasing sponsor payments.
Here are some additional resources that might be helpful:

[Github Sponsorships FAQs](https://event-sponsorship.github.com/faq/)
[Github Sponsorships documentation](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/paying-for-github-sponsors-by-invoice)

**Comment by leandrofars** [comment]
Thank you @AidenJamesYt , I've generated a ticket for github.

**Reply by AidenJamesYt** [reply]
np :D

glad i could help

**Comment by DHayes10** [comment, marked as answer]
Hi @leandrofars, 

Thanks for getting in touch. 

A payout of R$ 94.50 should now appear on your bank account statement as of 12/19/23, 12:15:10 AM. 

Best, 
GitHub Sponsors

**Reply by leandrofars** [reply]
Hi, I've received the payment. Thanks for the support and the quick resolution!
